extensions/azurecore/src/azureResource/utils.ts

```diff
@@ -376,7 +376,7 @@ export async function makeHttpRequest(account: azdata.Account, subscription: azu
 			break;
 	}
 
-	if (response.status >= 400) {
+	if (response.status >= 300) {
```

Comment:
and <200?

Reply:
https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#1xx_informational_response